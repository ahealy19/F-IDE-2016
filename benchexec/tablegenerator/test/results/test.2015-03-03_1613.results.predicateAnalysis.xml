<?xml version="1.0" ?>
<result benchmarkname="test" cpuCores="1" date="2015-03-03 16:13:02 CET" memlimit="3000 MB" name="predicateAnalysis" options="-noout -setprop log.consoleLevel=WARNING -predicateAnalysis" timelimit="10 s" tool="CPAchecker" toolmodule="benchexec.tools.cpachecker" version="1.4-svn 15944M">
  <columns>
    <column title="status"/>
    <column title="cputime"/>
    <column title="walltime"/>
  </columns>
  <systeminfo hostname="tortuga">
    <os name="Linux 3.13.0-45-generic x86_64"/>
    <cpu cores="8" frequency="3401 MHz" model="Intel Core i7-2600 CPU @ 3.40GHz"/>
    <ram size="16389384 kB"/>
  </systeminfo>
  <run files="[test/programs/simple/bitvectors/implicitunsignedconversion_false-unreach-label.i]" name="test/programs/simple/bitvectors/implicitunsignedconversion_false-unreach-label.i" properties="unreach-label">
    <column title="status" value="true"/>
    <column title="cputime" value="2.169189428s"/>
    <column title="walltime" value="2.1783649921417236s"/>
    <column hidden="true" title="category" value="wrong"/>
    <column hidden="true" title="exitcode" value="0"/>
    <column title="memUsage" value="128032768"/>
  </run>
  <run files="[test/programs/simple/switch_test_default_fallthrough_false-unreach-label.c]" name="test/programs/simple/switch_test_default_fallthrough_false-unreach-label.c" properties="unreach-label">
    <column title="status" value="false(reach)"/>
    <column title="cputime" value="2.189887353s"/>
    <column title="walltime" value="2.1968865394592285s"/>
    <column hidden="true" title="category" value="correct"/>
    <column hidden="true" title="exitcode" value="0"/>
    <column title="memUsage" value="129118208"/>
  </run>
  <run files="[test/programs/simple/compoundLiteral_true-unreach-label.c]" name="test/programs/simple/compoundLiteral_true-unreach-label.c" properties="unreach-label">
    <column title="status" value="true"/>
    <column title="cputime" value="2.22954344s"/>
    <column title="walltime" value="2.237081289291382s"/>
    <column hidden="true" title="category" value="correct"/>
    <column hidden="true" title="exitcode" value="0"/>
    <column title="memUsage" value="132337664"/>
  </run>
  <run files="[doc/examples/example.c]" name="doc/examples/example.c">
    <column title="status" value="true"/>
    <column title="cputime" value="2.272238619s"/>
    <column title="walltime" value="2.2822041511535645s"/>
    <column hidden="true" title="category" value="missing"/>
    <column hidden="true" title="exitcode" value="0"/>
    <column title="memUsage" value="129470464"/>
  </run>
  <run files="[test/programs/simple/builtin_expect_true-unreach-label.c]" name="test/programs/simple/builtin_expect_true-unreach-label.c" properties="unreach-label">
    <column title="status" value="true"/>
    <column title="cputime" value="2.058647067s"/>
    <column title="walltime" value="2.0664572715759277s"/>
    <column hidden="true" title="category" value="correct"/>
    <column hidden="true" title="exitcode" value="0"/>
    <column title="memUsage" value="126312448"/>
  </run>
  <column title="cputime" value="10.933401000000002"/>
  <column title="walltime" value="11.626914262771606"/>
</result>
