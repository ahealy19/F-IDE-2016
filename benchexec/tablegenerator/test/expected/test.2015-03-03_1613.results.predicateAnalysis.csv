tool		CPAchecker 1.4-svn 15944M	CPAchecker 1.4-svn 15944M	CPAchecker 1.4-svn 15944M	CPAchecker 1.4-svn 15944M
run set		predicateAnalysis	predicateAnalysis	predicateAnalysis	predicateAnalysis
		status	cputime (s)	walltime (s)	memUsage
test/programs/simple/bitvectors/implicitunsignedconversion_false-unreach-label.i	unreach-label	true	2.169189428	2.1783649921417236	128032768
test/programs/simple/switch_test_default_fallthrough_false-unreach-label.c	unreach-label	false(reach)	2.189887353	2.1968865394592285	129118208
test/programs/simple/compoundLiteral_true-unreach-label.c	unreach-label	true	2.22954344	2.237081289291382	132337664
doc/examples/example.c		true	2.272238619	2.2822041511535645	129470464
test/programs/simple/builtin_expect_true-unreach-label.c	unreach-label	true	2.058647067	2.0664572715759277	126312448